tck/README.adoc

```diff
@@ -404,77 +404,44 @@ following in your pom.xml:
                 <configuration>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
                     <suiteXmlFiles>
-                        <suiteXmlFile>tck-base-suite.xml</suiteXmlFile>
+                        <suiteXmlFile>${project.build.directory}/tck-suite/suites/tck-build-suite.xml</suiteXmlFile>
```

Comment:
`tck-base-suite.xml` is only run during the TCK build here in the MP JWT project, and those tests are also in the basic suite so it does not need to be included